### PR TITLE
fix(deps): upgrade tools and mod packages

### PR DESCRIPTION
**Pull request by mchill**
I encountered issue https://github.com/marwan-at-work/mod/issues/9 in a project that uses Go 1.18. The project uses Renovate's [gomodUpdateImportPaths](https://docs.renovatebot.com/configuration-options/#postupdateoptions) option, which output this error:

```bash
Command failed: mod upgrade --mod-name=github.com/go-playground/validator -t=10
2022/04/18 19:10:55 internal error: package "fmt" without types was imported from "gitlab.com/..."
```

I tested upgrading the tools package to the latest version locally, based on the solution the last time this issue was brought up, and the above command worked for my project.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/google/go-querystring v0.0.0-20170111101155-53e6ce116135 // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/urfave/cli/v2 v2.3.0
-	golang.org/x/mod v0.2.0
+	golang.org/x/mod v0.6.0-dev.0.20220106191415-9b9b3d81d5e3
```
Comment (marwan-at-work) [inline go.mod:11]: any reason not to use the latest stable which is v0.5.1 https://pkg.go.dev/golang.org/x/mod ?

**Review by marwan-at-work — COMMENTED** [review]
Thanks for the contribution, just one comment

File: go.mod (hunk 1 above)
Comment (mchill) [reply go.mod:11]: It was bumped automatically as a dependency of tools. https://github.com/golang/tools/blob/v0.1.10/go.mod#L7

tools 0.1.9 uses mod 0.5.1. I tested that with my project and it still works. I can downgrade it if you want to use the stable version.

Comment (marwan-at-work) [reply go.mod:11]: I see. Will merge then, thanks!

**Pull request merged**